addons/actions/src/preview/__tests__/actions.test.js

```diff
@@ -0,0 +1,90 @@
+import addons from '@storybook/addons';
+import { actions } from '../..';
```

Comment:
I feel a lot better having a test of some of the new logic. Not sure if this is the best factoring. Some prior art from the action.test. Could perhaps make it a little less verbose but...